cpu/native/socket_zep/socket_zep.c

```diff
@@ -304,21 +304,8 @@ static int _init(netdev_t *netdev)
 
 static int _get(netdev_t *netdev, netopt_t opt, void *value, size_t max_len)
 {
-    socket_zep_t *dev = (socket_zep_t *)netdev;
-    uint16_t *v = value;
-
-    assert((dev != NULL));
-    switch (opt) {
-        case NETOPT_MAX_PACKET_SIZE:
-            assert(value != NULL);
-            if (max_len != sizeof(uint16_t)) {
-                return -EOVERFLOW;
-            }
-            *v = SOCKET_ZEP_FRAME_PAYLOAD_LEN;
-            return sizeof(uint16_t);
-        default:
-            return netdev_ieee802154_get(&dev->netdev, opt, value, max_len);
-    }
+    assert(netdev != NULL);
+    return netdev_ieee802154_get(netdev, opt, value, max_len);
```

Comment:
Why not remove this function alltogether and set the `.get` [here] to `netdev_ieee802154_get`?

[here]: https://github.com/RIOT-OS/RIOT/blob/9b1d91202fd5d5d77cb0e386b80395a51ad114ea/cpu/native/socket_zep/socket_zep.c#L324

Reply:
I'd say for two reasons, because i kept it similar to the _set function but mainly because the `assert(netdev != NULL);` is not done in `netdev_ieee802154_get`.

Reply:
Good enough for me, I can also imagine that we're going to need this function in the future again.

Reply:
I just want to test if everything still works at runtime. Changes still look good to me.